clearwater-infrastructure/usr/share/clearwater/infrastructure/scripts/config_validation/project_clearwater_options.py

```diff
@@ -0,0 +1,154 @@
+#
```

Comment:
Why is this called project_clearwater_options?

Reply:
As opposed to clearwater core options, which may be defined in a separate file - it may be better to cross that bridge when we get to it and name this something clearer though.